README.md

```diff
@@ -1,67 +1,146 @@
-
-Slash commands are only included in DiscordJS v13.x.x or higher. For more information on slash (/) commands and registering them see [Discord's documentation](https://discordjs.guide/interactions/registering-slash-commands.html#guild-commands).
-
----
-
-### ideas/todo
-From previous Python bot ->
-1. accrue tokens with activity
-2. distribute tokens person to person
-3. records log of every bet and distribution of tokens
-4. *salsa*
-5. register different kinds of bets (rng, bets regarding users, bets regarding collectives)
-6. **guac**
-7. algorithm that determines odds payouts based on bet sizes for each outcome, or even use data to determine likelihood of an event (use porofessor to retrieve winrates, etc)
-8. distribute base amount of tokens
-9. cash in tokens for discord/gaming privileges/punishments
-10. roll_deez
-11. roll_meez_shinz
-12. quotes
-13. ask for quote of specific user
-14. after for random or numbered quote
-
-From after creation of this bot ->
-1. consolidate audio and voice connect related code within `utils/connectAndPlayAudioFile.js` and `utils/speakText`
-2. set up music bot capabilites with queueing, pausing, stopping, etc. 
-    - potentially take all music bot code and put into a separate bot so as to avoid voice-overlap. alternatively, have the bot not do anything voice related while playing audio/music.
-3. set audio recording capabilities
-4. add constants.js with constant definitions
-5. change /playaudio and /listaudio to /audio play and /audio list subcommands
\ No newline at end of file
+# PASTRAMI BOT
+
+[![GitHub issues](https://img.shields.io/github/issues/jacobc18/js-discord-bot)](https://github.com/jacobc18/js-discord-bot/issues)
+
+![hello there](https://media.giphy.com/media/3ornk57KwDXf81rjWM/giphy.gif)
+
+
+
+
+Pastrami is a multifunctional JavaScript Discord bot with abilities including
+
+- TTS greetings for voice channel user join
+- Audio streaming via YouTube links 
+- Information on famous bald people
+
+## Installation
+
+Requires [NodeJS v16.x.x](https://nodejs.org/en/) or higher and the latest version of [NPM](https://docs.npmjs.com/cli/v7/commands/npm-install)
+
+### Windows
+
+```
+$ windows_setup.bat
+```
+
+### Manual
+
+In package directory run
+
+```
+$ npm install
+```
+
+## Setup
+
+In the repo folder create a file called ``` .env ``` with the contents
+
+```
+DISCORD_TOKEN=your-discord-bot-token-goes-here
+CLIENT_ID=your-discord-client-id-goes-here
+```
+
+To get a Discord bot token and Discord client ID see [How to Make a Discord Bot in the Developer Portal](https://realpython.com/how-to-make-a-discord-bot-python/#how-to-make-a-discord-bot-in-the-developer-portal).
+
+We utilize [say.js](https://github.com/Marak/say.js) to convert text strings to mp3 files. Say.js uses some [FFMpeg](https://www.ffmpeg.org/) libraries to operate, so you will need to install the most up to date version. You can learn how to do that [here](https://www.wikihow.com/Install-FFmpeg-on-Windows). 

```

Comment:
I would suggest avoiding 'we', 'us', 'I', etc. in docs like this. Use Pastrami/PastramiBot instead here: 'We utilize ...' -> 'Pastrami utilizes ...'